opencga-app/src/main/java/org/opencb/opencga/app/cli/CommandExecutor.java

```diff
@@ -367,15 +370,15 @@ private String printParameterValue(RestParameter parameter) {
             case "long":
                 return "0";
```

Comment:
What are you expecting to return here? `"0"` string or `0` number?
I think that `printParameterValue` should return "Object" instead of "String"

Reply:
This method is to return the json value field, it does not return the object, it is always a String